### DISCUSSION TITLE
Is it possible to increase the count of  userlong in custom statistics?

### DISCUSSION THREAD

**Discussion by kdw9502** [naver/ngrinder#729] [Q&A] [answered]
According to the [grinder's guide](http://grinder.sourceforge.net/g3/script-javadoc/net/grinder/script/Statistics.html), we can only use 5 long and 5 double for custom statistic. 

I'm testing native library and I need more than 10 variables for custom statistics. Is there a solution to this?

Or Is there an easy way to modify and build grinder.jar for this?

**Comment by donggyu04** [comment, marked as answer]
@kdw9502 

**Unfortunately, It is impossible.** But, I sent a patched version of ngrinder to your email(kdw9502@gmail.com)
You can use `userLong0 ~ userLong14, userDouble0 ~ userDouble14`

> You can simply run ngrinder with `java -jar ngrinder-controller-xxx.war`

**Reply by junoyoon** [reply]
@donggyu04 no.. don't do that.

**Reply by donggyu04** [reply]
@junoyoon 
oops.. I already sent the file. is there any problem?

**Reply by junoyoon** [reply]
Please make it as a feature if the request is reasonable or drop the request. it's better way to make ngrinder maintainable.

**Reply by donggyu04** [reply]
@junoyoon 

Yes, I will be careful next time :)

**Reply by kdw9502** [reply]
Can I request it as feature for future release?

**Reply by donggyu04** [reply]
@kdw9502 

Yes, I'll consider a efficient way to support it.

**Reply by kdw9502** [reply]
I'm struggling with this and issue #718. How can I fix code and build it like #718?

**Reply by donggyu04** [reply]
@kdw9502 

Couldn't you fix & build the project yet? or you have been fixed the project but It doesn't work you want?

**Reply by kdw9502** [reply]
I want to disable TPS_TOO_LOW error and to increase the count of userLong both. I can fix former one but not with latter one. Can you see where to modify the code to increase the number of userLongs?

**Reply by junoyoon** [reply]
What make your test emit TPS_TOO_LOW? 
Usually TPS_TOO_LOW happens when your target server returns no response or .. you record wrong method.

**Reply by junoyoon** [reply]
By the way.. please use  the different thread of different question. We won't answer the question in this comments any more.
